src/transaction.js

```diff
@@ -41,6 +41,14 @@ export class Transaction extends Transform {
     this.meta = Object.create(null)
   }
 
+  // :: ((Transaction, ...args: *)) → Transaction
```

Comment:
Wasn't sure if we needed to specify a return type of `*` for the argument function